ls.nim

```diff
@@ -130,7 +130,7 @@ proc orCancelled*[T](fut: Future[T], ls: LanguageServer, id: int): Future[T] {.a
       raise fut.error
   else:
     debug "Future cancelled.", id = id
-    let ex = newException(Cancelled, fmt "Cancelled {id}")
+    let ex = newException(CancelledError, fmt "Cancelled {id}")
```

Comment:
this _most likely_ will not work - ie cancellation in chronos is a multi-step affair that cancels dependent futures - most likely, `orCancelled` must be replaced entirely to use the native chronos cancellation flow - until then, it's best not to raise CancelledError "manually" (this is something only chronos should do) - ie better to use the current exception